### PR TITLE
Add accessibility to the portfolio.

### PR DESCRIPTION
**Pull request by aubinleyoung**
## Points that we have checked

- [ ] Page titles are shown in the window title bar in the browser.

- [ ] Image text alternatives: added descriptive alt to images
- [ ] Text headings: corrected the hierarchy of heading levels
- [ ] Color contrast: There was sufficient color contrast.
- [ ] Resize: no issues found.
- [ ] Interaction: We added labels to the form input tags.
- [ ] Moving content: no issues found.
- [ ] Multimedia: no issues found.
- [ ] The basic structure of the page: no issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,177 +1,264 @@
...
+          sign-upsrequired. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer tooka standard dummy text.
+        </p>
+        <ul class="tools-container">
+          <li class="tools">html</li>
+          <li class="tools">css</li>
+          <li class="tools">bootstrap</li>
+          <li class="tools">Ruby</li>
+        </ul>
+        <button class="see-project-btn" type="button">See Project</button>
```
Comment (rskinnerc) [inline index.html:85]: ♻️ Your project is going well so far. Please note though, that when checking the accessibility score, I found this issue with the contrast of most of your buttons. The low-contrast text might make it difficult for some users to read. 

image

- [ ] Please double-check the buttons' text color contrast to make it readable.

File: index.html [hunk 2]
```diff
@@ -1,177 +1,264 @@
...
+      </ul>
+      <li class="cat-title-1-3">Skills</li>
+      <ul class="cat-title-2 grid3">
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </ul>
```
Comment (rskinnerc) [inline index.html:216]: ♻️ Well done using lists to structure your skills/stack! However, as part of the accessibility checks, the basic structure of the page should be navigable. Please note how a `ul` should contain `li` items only. That way, the screen readers would be able to navigate the page correctly.

image

- [ ] Kidnly update your list so the inner list (the inner `ul` tag) is placed inside another `li` tag. That is, the inner list should be another item of the outer list.

**Review by rskinnerc — CHANGES_REQUESTED** [review]
Hi @aubinleyoung and @ambrose-kibet 👋👋👋👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy

## Highlights
- Good job using GitHub Flow
- Linters correctly configured and passing ✅
- You are using the required checklist for accessibility

## Required Changes ♻️
_Check the comments under the review._
Great job using the recommended list! However, I have pointed out some changes that would make your portfolio even more accessible. you could get some hints on how accessible your page is using Lighthouse from Chrome dev tools:

image

Kindly, please consider making the required changes. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (OyePriscilla) [reply index.html:85]: - [ ] Please kindly implement this required change, you might want to use this [tool](https://www.aditus.io/button-contrast-checker/) to balance contrast. 👍 Kindly run your live demo through it and consider making all the changes suggested by the tool.

**Review by Owusu-Desmond — APPROVED** [review]
Hi @aubinleyoung , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**